calibration/calibrate_model.sh

```diff
@@ -183,12 +183,17 @@ if [[ -n $LIMIT ]]; then
     EXTRA_FLAGS_STEP_1+="--max-dataset-samples $LIMIT "
 fi
 
+SKIP_STEP_1=false
 if  [[ "$model_name_lower" == *"deepseek"* ]]; then
```

Comment:
Can we enable pile dataset processing for other models too if the `DATASET_PATH` does not end with `.pkl`? I don't think the default dataset processed by LLaMA tokenizer is a suitable choice for all other models as their vocab sizes are different.